### DISCUSSION TITLE
Is yabai working on Mojave possible?

### DISCUSSION THREAD

**Discussion by VicDobrov** [asmvik/yabai#1505] [Ideas]
I would like to launch yabai on Mojave. This system works great on a variety of computers.
What is needed for this? Finances?

**Comment by asmvik** [comment]
I'm not interested in trying to maintain yabai for older versions of macOS that I do not have the ability to test personally, and that Apple no longer supports, sorry.

The last working version for Mojave would be v4.0.4. There are a couple of fixes that you probably want to cherry-pick from upstream if you decide to run this version. You can pull the repo and checkout the v4.0.4 tag to apply changes manually and build from source.

The changes that are of interest would be these:


- Fix regression causing a window to be moved to the active space of an inactive display when sent to an inactive space of an inactive display https://github.com/koekeishiya/yabai/issues/1053
- Fix regression causing hidden windows to not show in window queries https://github.com/koekeishiya/yabai/issues/1421
- Clamp split ratio instead of resetting when an out of range / invalid value is given https://github.com/koekeishiya/yabai/pull/1401
- Applying rule with property manage=on would cause both minimized and hidden windows to be managed, even though the window is not visible https://github.com/koekeishiya/yabai/issues/1418
- Fix regression causing window sticky to not work properly https://github.com/koekeishiya/yabai/issues/1424
- Properly update window ordering when a window is added to the top of a stack https://github.com/koekeishiya/yabai/issues/1311
- Properly update insertion point in a window stack when the marked window is removed from the stack https://github.com/koekeishiya/yabai/issues/1275


Edit: I am not aware of how much of the functionality actually works and doesn't work on Mojave. I have not been able to test on Mojave for years at this point.
